### PR TITLE
Use systemd to manage kismatic-inspector

### PR DESCRIPTION
**Pull request by dkoshkin**
The Kismatic Inspector didn't always start correctly and caused many integration test failures

Minor fixes:
* Print Ansible run command in a new line 
* Always respect `LEAVE_ARTIFACTS`

### CODE REVIEW

File: ansible/roles/preflight/tasks/main.yaml [hunk 1]
```diff
@@ -3,32 +3,31 @@
...
       mode: 0744
-  - name: start the Kismatic Inspector in server mode
-    command: "{{ bin_dir }}/kismatic-inspector server --node-roles {{ \",\".join(group_names) }} --port 8888 {{ (allow_package_installation == 'true') | ternary('', '-e') }}"
-    async: 300
-    register: start_inspector
-  - name: ensure Kismatic Inspector server is running
-    async_status: jid={{ start_inspector.ansible_job_id }}
-    register: job_result
+  - debug: msg="{{ bin_dir }}/kismatic-inspector server --node-roles {{ \",\".join(group_names) }} --port 8888 {{ (allow_package_installation == 'true') | ternary('', '-e') }}"
+  - name: copy calico-node.service to remote
```
Comment (alexbrand) [inline ansible/roles/preflight/tasks/main.yaml]: Name is wrong here

**Comment by dkoshkin** [comment]
I though about adding it, you only need that when the service is already running and the file gets changed.
I'll add it because that could happen, ie. when a node changes roles

**Comment by alexbrand** [comment]
Sounds good. Other than that LGTM

**Comment by dasmb** [comment]
LGTM.

**Pull request merged**